user-console/src/main/java/org/pentaho/mantle/client/solutionbrowser/fileproperties/FilePropertiesDialog.java

```diff
@@ -216,6 +218,14 @@ public void showTab( Tabs tab ) {
       }
     }
   }
+  @Override
+  public void center() {
+    super.center();
+    Focusable focusable = ElementUtils.findFirstKeyboardFocusableDescendant( this.propertyTabs );
+    if ( focusable != null ) {
+      focusable.setFocus( true );
+    }
+  }
```

Comment:
Can you explain why this is needed? Why does the existing generic code fail?
Also, _if_ you'd really need to do this, ideally, you'd do it before super.center and by passing the intended widget to `setFocusWidget(.)`.

Reply:
Please see https://github.com/pentaho/pentaho-commons-gwt-modules/pull/843 which addresses this bug.

Reply:
I tried suggested change but it would break the behaviour so I have kept it as is until further instruction.